sample/src/main/res/layout/group_map_and_movement.xml

```diff
@@ -11,6 +11,27 @@
         style="@style/ButtonCommon"
         android:text="Current Floor" />
 
+    <Button
+        android:id="@+id/btnNewFloor"
+        style="@style/ButtonCommon"
+        android:text="New Floor" />
+
+    <Button
+        android:id="@+id/btnDeleteFloor"
+        style="@style/ButtonCommon"
+        android:text="Delete Floor" />
+
+    <Button
+        android:id="@+id/btnRenameFloor"
+        style="@style/ButtonCommon"
+        android:text="Rename Floor" />
+
+    <Button
+        android:id="@+id/btnGetFloorData"
+        style="@style/ButtonCommon"
+        android:text="Get Floor Data" />
```

Comment:
Multi-floor methods should not be this obvious in sample app, as they are very advanced functionalities. Let's put them in a deep rows or nested pages.